apps/search/js/providers/suggestions.js

```diff
@@ -44,8 +52,6 @@
 
     name: 'Suggestions',
 
-    remote: true,
```

Comment:
Whether we search is dependant on whether its a url, a private browsing instance and whether searches are enabled. I can / have a patch to set a Search.preventRemoteFetch, however that is far less readable than the `boolean trap`

`remote: true` is removed because when suggestions are disabled the provider still needs to be called in order to render a default

We can possibly go down the 'make the single link a different provider' path again but there are issues with the shared logic (how it clears the containers / hides the heading) and I think its ultimately a more confusing than the current way